File: javascript01/step11/test02_3.js
/* DBMS에 SQL명령 보내기 3) update : 데이터 갱신 명령
*/

const mysql = require('mysql')

var connection = mysql.createConnection({
  user     : 'webapp',
  password : '1111',
  database : 'webappdb'
})

connection.connect()

// name이 aaa인 사람의 tel을 바꿔라
connection.query('update memb set tel="222-2222" where name="aaa"', function (error, results) {
  console.log('결과를 가져왔음')
  if (error) throw error

  console.log(results)
})

connection.end()
console.log('MySQL 서버와의 연결 끊기가 예약되었음')

File: java01/src/step20/Test02_3.java
/* JDBC 프로그래밍: Statement 객체 - UPDATE 실행
 */
package step20;

import java.sql.DriverManager;

public class Test02_3 {

  public static void main(String[] args) throws Exception {

    com.mysql.jdbc.Driver mysqlDriver = new com.mysql.jdbc.Driver();

    DriverManager.registerDriver(mysqlDriver);

    java.sql.Connection con = DriverManager.getConnection(
        "jdbc:mysql://localhost:3306/webappdb", 
        "webapp", 
        "1111"); 

    java.sql.Statement stmt = con.createStatement();

    int count = stmt.executeUpdate("update memb set name='호호호' where email='hong100@text.com'");
    
    System.out.println(count);

    stmt.close();

    con.close();

  }

}





